Sources/Props/ProgressView/BarsProgressStyle.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  BarsProgressStyle.swift
+//  
+//
+//  Created by Djuro on 1/24/22.
+//
+
+import SwiftUI
+
+public struct BarsProgressStyle: ProgressViewStyle {
+    
+    // MARK: - Initializer
+    public init() {}
+    
+    // MARK: - ProgressViewStyle
+    public func makeBody(configuration: Configuration) -> some View {
+        IndeterminateView(count: 8,
+                          spacing: 8,
+                          cornerRadius: 8,
+                          scaleRange: 0.5...1,
+                          opacityRange: 0.25...1)
+            .frame(width: 200, height: 50)
+    }
+    
+    struct IndeterminateView: View {
+        
+        // MARK: - Properties
+        @State var isOn = false
+        let count: Int
+        let spacing: CGFloat
+        let cornerRadius: CGFloat
+        let scaleRange: ClosedRange<Double>
+        let opacityRange: ClosedRange<Double>
+        private var scale: CGFloat { CGFloat(isOn ? scaleRange.lowerBound : scaleRange.upperBound) }
+        private var opacity: Double { isOn ? opacityRange.lowerBound : opacityRange.upperBound }
+        
+        var body: some View {
+            GeometryReader { geometry in
```

Comment:
You probably don't need to pull out GeometryReader here and can let SwiftUI take this with an HStack and Spacers. Maybe that'd help with the portrait orientation issue? Might be unrelated. You're having to do some math to compute frames, but I suspect that SwiftUI would be happy to take that some of that off your hands.